HARK/ConsumptionSaving/tests/test_PerfForesightConsumerType.py

```diff
@@ -65,19 +65,20 @@ def test_simulation(self):
         )  # This implicitly uses the assign_parameters method of AgentType
 
         # Create PFexample object
-        self.agent_infinite.track_vars = ["mNrm"]
+        self.agent_infinite.track_vars = ["bNrm", "mNrm", "TranShk"]
         self.agent_infinite.initialize_sim()
         self.agent_infinite.simulate()
 
         self.assertAlmostEqual(
```

Comment:
Another one of those we *might* keep in with a high tolerance

Reply:
What I've done here (just to mix it up...) is keep the test for the moment (the mean) but turned it into a test of the transition function.